client/src/pages/Lobby.js

```diff
@@ -97,6 +97,30 @@ const Lobby = () => {
 
   if (!open) return <Redirect to='/dashboard' />;
 
+  const addUserAndQuestions = async () => {
+    if (Object.keys(userData).length === 1) {
+      await addInterviewQuestions(roomId);
+    } else {
+      if (creatorId) {
+        userData.forEach(async (user) => {
+          if (user._id !== creatorId) {
+            await addUserToInterview({ userId: user._id, roomId });
+            await addInterviewQuestions(roomId);
```

Comment:
I'm a bit confused by the flow here. Wouldn't we only start the interview if both users are present? Wouldn't you only want to addInterviewQuestions once?